frontend/packages/local-storage-operator-plugin/src/components/disks-list/disks-list-page.tsx

```diff
@@ -190,7 +218,7 @@ export const NodesDisksListPage: React.FC<NodesDisksListPageProps> = ({
           selector: { [LABEL_SELECTOR]: nodeName },
         },
       ]}
-      customData={{ node: nodeName, EmptyMsg }}
+      customData={{ node: nodeName, EmptyMsg, errorMessage }}
```

Comment:
Lets combine all errors and loading state. These all can be handled by table component.
And remove any other loading states.
```suggestion
      customData={{
        node: nodeName,
        EmptyMsg,
        error: subscriptionLoadError || lvdRequestError,
        loaded: subscriptionLoaded,
      }}
```

Reply:
@afreen23 There doesn't seem to be a variable named `subscriptionLoadError` defined, also there is no `loadError` property on the `subscription` variable as its not `const subscription: FirehoseResult<K8sResourceKind[]> | FirehoseResult<K8sResourceKind>`, but `const subscription: SubscriptionKind[]`. Should I just use `lvdRequestError` here?

Reply:
We need to add that `subscriptionLoadError` . For watchResources hook you can extract that.